components/service/glean/docs/pings.md

```diff
@@ -0,0 +1,42 @@
+# glean pings
+
+glean provides a set of pings that are assembled, out of the box, without any developer intervention.
```

Comment:
Should we highlight again that this is only when collection is enabled?

Reply:
Good point! I've slightly rephrased this.